setup.cfg

```diff
@@ -0,0 +1,4 @@
+[pep8]
```

Comment:
One of the next things I'm planning on doing is to set up Travis CI to automatically run a pep8 check every time someone checks in code.